build.gradle

```diff
@@ -142,4 +142,25 @@ subprojects {
         outputFormat = 'javadoc'
         outputDirectory = "$buildDir/javadoc"
     }
+
+    task sourcesJar(type: Jar, dependsOn: classes) {
```

Comment:
Why does this depend on compiling the classes? Shouldn't we be able to package the sources right away?

Reply:
Correct, done.